packages/serve/index.ts

```diff
@@ -170,7 +170,7 @@ function serve() {
 	}
 }
 
-export = {
+export {
```

Comment:
we actually don't need these exports. Safe to remove

Reply:
do this and I'll do a lgtm

Reply:
We should keep `serve` in case someone uses this method: https://github.com/webpack/webpack-cli/tree/master/packages/serve

Reply:
You can do a default export of serve only, adjust the docs to use default export:

` const serve = require('rick-n-morty/serve'=`

Reply:
Oh wait then it would be `const serve = require("@webpack-cli/serve").default;`, seems kind of ugly compared to `.serve`.

Reply:
Maybe you could do the same as in `init` ?

https://github.com/webpack/webpack-cli/tree/master/packages/init#node